src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj

```diff
@@ -56,7 +56,7 @@
     <Reference Include="System.Net.Http" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Client.NativeInterop" Version="0.12.1" />
+    <PackageReference Include="Microsoft.Identity.Client.NativeInterop" Version="0.12.2" />
```

Comment:
Any value in adding a wildcard version?

Reply:
unfortunately no because of [issues like this](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3623#discussion_r952485293). So until we finalize the interop we have to be on specific versioning.